anaconda_project/archiver.py

```diff
@@ -215,9 +218,9 @@ def _enumerate_archive_files(project_directory, frontend, requirements):
     if git_filter is None or ignore_file_filter is None:
         return None
 
-    plugin_patterns = set()
+    plugin_patterns = {'/anaconda-project-local.yml'}
```

Comment:
With this change it is no longer necessary to include this file in projectignore